README.md

```diff
@@ -437,21 +436,15 @@ jobs:
 
 1. The regular expression in step **Check tag** should be:
 
-```
-^refs/tags/v[0-9]{1,3}.[0-9]{1,3}.[0-9]{1,3}$
-```
+    ^refs/tags/v[0-9]{1,3}.[0-9]{1,3}.[0-9]{1,3}$
```

Comment:
This way, the code does no longer have syntax highlighting. Same for the below two cases.

Reply:
Okay. So I do have to fix it the other way around and using fences everywhere. That's btw what I'm always doing but in this document indention was used in the first place.